README.md

```diff
@@ -614,7 +614,16 @@ You can read more about it in [this article](https://netbasal.com/optimize-user-
 
 ## Unit Testing
 
-When running specs, we want the have the languages available immediately, in a synchronous fashion. Transloco provides you with a `TranslocoTestingModule`, where you can pass the languages you need in your specs. For example:
+When running specs, we want to have the languages available immediately, in a synchronous fashion. To do so ensure you've correctly configured the TypeScript compiler by adding the following properties in `tsconfig.json`.
```

Comment:
Move your section below the code, and add: "Note that in order to import JSON files, you need to configure the TypeScript compiler by adding the following properties in `tsconfig.json`:
...